db/migrations/20170830183100_add_key_label_column_to_tables_with_encrypted_columns.rb

```diff
@@ -0,0 +1,32 @@
+Sequel.migration do
+  up do
+    tables_to_alter = %w(
+      env_groups
+      service_brokers
+      service_bindings
+      service_instances
+      service_keys
+      apps
+      buildpack_lifecycle_buildpacks
+      buildpack_lifecycle_data
+      droplets
+      packages
+      tasks
+    )
+    tables_to_alter.each do |table|
+      alter_table(table.to_sym) do
+        add_column :key_label, String
```

Comment:
We were thinking that the name `key_label` on its own might not be very meaningful to future devs (and current devs not familiar with this work).  Maybe we prefix it with `encryption_`? So the column would be `encryption_key_label` or `encryption_key_used` or something even better of your own choosing 😸 .

Reply:
@tcdowney Yeah, totally reasonable.

Reply:
@elenasharma So you had an existing deployment, you applied the PR as a patch, then pushed?  Roughly correct?  Just want to make sure I test the right scenario.

Reply:
@elenasharma @tcdowney I know what's going on.  We bumped the cipher length from 128 to 256, but we didn't test for stored 8 byte salt values (256 requires an IV of 16 bytes), which is what you ran into.  I think that, for the time being, we'll back off to 128, then we can figure out a way to move to AES-256 down the road (which we do want to do).

Reply:
Nope.  Using the correct method for computing the key (what we're doing now, instead of the old way does require a 16 bit iv, unfortunately.  We'll have to think of a way to handle legacy data.

Reply:
Got it fixed.